README.markdown

```diff
@@ -115,6 +131,30 @@ One can apply a "sort" constraint, which analyzes documents in the specified ord
 
     $ mongo test --eval "var collection = 'users', sort = { updated_at : -1 }" variety.js
 
+### Output formats ###
+
+Variety supports two different output formats:
+
+- ASCII: nice formatted tables (as in this readme)
```

Comment:
Instead of "nice formatted tables" the correct English would be "nicely formatted tables".